torch/fx/proxy.py

```diff
@@ -334,6 +334,9 @@ def find_tracer(a):
         if torch.overrides.is_tensor_method_or_property(orig_method):
             return tracer.create_proxy('call_method', orig_method.__name__, args, kwargs)
         else:
+            if orig_method.__name__ in {"cond", "map"}:
```

Comment:
@angelayi are you sure this way of testing won't accidentally catch other function with the same name? could we add one unittest?

Reply:
Updated to now just check for pyoperators!